Cargo.toml

```diff
@@ -28,7 +28,7 @@ lazy_static = { version = "~1", optional = true }
 log = "~0.3.8"
 lru_time_cache = "~0.8.1"
 maidsafe_utilities = "~0.18.0"
-mock-quic-p2p = { git = "https://github.com/madadam/quic-p2p", branch="mock", optional = true }
```

Comment:
Could you update the commit that refer to madadam instead, probably best to avoid having commit refering to external repo.